### PR TITLE
Events added to Secondary

### LINKED ISSUE #1413: Add events to Secondary
We should have an event in `Secondary`'s constructor and `transferPrimary` functions, indicating the new primary address. I'm not sure about naming, for consistency with `Ownable` it should be something like `PrimarinessTransferred`, but that sounds awful. Ideas are welcome!

While we're at it, we should also improve this contract's documentation.

### PR DESCRIPTION
**Pull request by Aniket-Engg**
Fixes #1413

### CODE REVIEW

File: contracts/ownership/Secondary.sol [hunk 1]
```diff
@@ -7,11 +7,16 @@ pragma solidity ^0.4.24;
 contract Secondary {
   address private _primary;
 
+  event PrimarinessTransferred(
```
Comment (nventuro) [inline contracts/ownership/Secondary.sol]: Can we rename this event to `PrimaryTransferred`?

File: contracts/ownership/Secondary.sol [hunk 2]
```diff
@@ -22,13 +27,20 @@ contract Secondary {
...
+  /**
+   * @return the address of the primary.
+   */
   function primary() public view returns (address) {
     return _primary;
   }
-
+  
+  /**
+   * @dev Transfers primariness to a recipient.
```
Comment (elopio) [inline contracts/ownership/Secondary.sol]: I think primariness is a terrible word X-)
Maybe: Transfers the contract to a new primary.

Comment (Aniket-Engg) [reply contracts/ownership/Secondary.sol]: updated it. Thanks!

**Comment by Aniket-Engg** [comment]
Costructor event tests are related to #1354

**Comment by nventuro** [comment]
Thanks @Aniket-Engg for pointing that out :) I created https://github.com/OpenZeppelin/openzeppelin-solidity/issues/1436 to track the whole thing.

**Pull request merged**